src/main.vala

```diff
@@ -25,6 +44,12 @@ public static int main (string[] args) {
     Environment.set_application_name (APP_NAME);
     Environment.set_prgname (APP_NAME);
 
+    Bus.own_name (BusType.SESSION, "org.freedesktop.FileManager1", BusNameOwnerFlags.REPLACE,
```

Comment:
Can we get this into the already available `dbus_register` of the `Marlin.Application` method?
The `GLib.Application` provides a way to register any needed objects in the application itself implementing the `dbus_register` and `dbus_unregister` methods like that:

```
public override bool dbus_register (DBusConnection connection, string object_path) throws Error {
    base.dbus_register (connection, object_path);

    // own the FileManager1 interface here
}
```

```
public override void dbus_unregister (DBusConnection connection, string object_path) {
    // Unown the FileManager1 interface here if it exists

    base.dbus_unregister (connection, object_path);
}